### DISCUSSION TITLE
Incorporate Frigate UI Metrics into HA?

### DISCUSSION THREAD

**Discussion by templeowls21** [blakeblackshear/frigate#15554] [Ask A Question]
Sorry if a dumb question....the Frigate UI in HA has several metrics it tracks (CPU Usage, storage capacity, etc.)

I was curious if it's possible to expose those as entities within HA. That way I can see them right on my HA dashboard and incorporate them into long term trending.

I know I can install tracking software onto my Frigate server to accomplish this but I thought I'd check if using the UI's metrics was possible first.

Thanks!

**Comment by NickM-27** [comment]
all of it is already available to HA, and many of them are already implemented as sensors that you just need to enable. Some things like storage metrics don't have sensors yet but this would not be difficult to implement

**Reply by templeowls21** [reply]
Sorry I should have noted...I'm not using the HA add-on. Frigate runs on a seperate server. I have the Frigate integration and the Frigate Proxy add-on. In the Frigate integration, there's a "frigate" entity, but below are the only diagnostics. I'm not seeing any sensors to enable
![Screenshot 2024-12-19 131314](https://github.com/user-attachments/assets/b2cc5142-7599-4aa2-b579-505684092088)

Any suggestions to track storage (sorry, this is all new to me)?

**Reply by NickM-27** [reply]
The camera metrics have many that are disabled by default.
